Jenkinsfile

```diff
@@ -116,8 +116,8 @@ pipeline {
     post {
         always {
             // Change ownership of the workspace to Jenkins for clean up.
-            withEnv(["HOME=${env.WORKSPACE}"]) {
-                sh 'chown -R 1003:1003 ${HOME}/'
+            withEnv(["WHOME=${env.WORKSPACE}"]) {
+                sh 'chown -R 1003:1003 ${WHOME}/'
```

Comment:
I believe this should be left as `HOME` (only in this final cleanup section). That is certainly what I was told and what I have done in my other packages (as I get to them).

Reply:
OK... I don't really understand why this is but that's ok. I thought it was a global change.
Thanks for the catch.